protos/client.proto

```diff
@@ -50,72 +50,80 @@ message ClientStateCurrentResponse {
     }
     Status status = 1;
     string merkle_root = 2;
-
 }
 
-// A request from a client for a particular address's data, for a given merkle root
-message ClientStateGetRequest {
+// A request to list every entry under a given merkle root or head block.
```

Comment:
You can specify either a head block, or a merkle root. Or neither, and the most current will be fetched. Could be clarified.